recipes/scijava-jupyter-kernel/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -euo
+
+# Install Beakerx...
+# ...waiting for a Maven artifact.
+cd $RECIPE_DIR/
+wget https://services.gradle.org/distributions/gradle-3.5-bin.zip
```

Comment:
I suggest using "curl" since wget is not in conda-forge. If you notice the build log, it pulled wget from "defaults".